webviz_subsurface/plugins/_reservoir_simulation_timeseries_onebyone.py

```diff
@@ -206,7 +206,7 @@ def ensemble_selector(self):
             style={"paddingBottom": "30px"},
             children=html.Label(
                 children=[
-                    html.Span("Ensemble", style={"font-weight": "bold"}),
+                    html.Span("Ensemble:", style={"font-weight": "bold"}),
```

Comment:
Not in scope of this PR?

Reply:
No, leftover from the "Clear change" I tested yesterday for the deploy test. Will move it to the PR on fixing the onebyone.